README.md

```diff
@@ -1,13 +1,21 @@
-# CS-01
+  CS-01
 An easy of Computer Programming with Real-life use-cases
 A Bytes Club initiative to improve Competitive Programming Culture
 
 ## Objectives
-
+   Our main objective is to implement the real life problems through computer science.
+  
 ## Course Structure
 
 ## Notifications
-
+   **Subscribe** to the mailing list of [Bytes Club](bytes-club@googlegroups.com)
```

Comment:
Link: https://groups.google.com/forum/#!forum/bytes-club (for joining request) or mailto:bytes-club@googlegroups.com (for mailing to the list)